### ISSUE TITLE
Consider creating a mode that works with remote server via http

### ISSUE THREAD

**Issue by jexp** [tmtxt/n4js.el#2]
That connects to the transactional rest endpoint and supports auth etc

**Comment by tmtxt** [comment]
I will check that to see whether I can use neo4j shell to connect to a remote http server (not very sure about this)

**Comment by jexp** [comment]
No, you would use http directly.
Or use something like https://github.com/nicolewhite/cycli

**Comment by tmtxt** [comment]
yeah cycli would be much better since it's can be integrated as an inferior shell, too and I don't have to rewrite the stuff that is already available
